composites/Plugin/Shared/components/KeywordInput.js

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import { FormattedMessage } from "react-intl";
+import styled from "styled-components";
+import colors from "../../../../style-guide/colors.json";
+import PropTypes from "prop-types";
+
+let KeywordField = styled.input`
+	margin-right: 0.5em;
+	border-color: ${ props => props.borderColor };
+`;
+
+const ErrorText = styled.div`
+	font-size: 1em;
+	color: ${ colors.$color_red};
+	margin: 1em 0;
+	min-height: 1.8em;
+`;
+
+class KeywordInput extends React.Component {
+	/**
+	 * Constructs a KeywordInput component
+	 *
+	 * @param {Object} props The props for this input field component.
+	 *
+	 * @returns {void}
+	 */
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			showErrorMessage: false,
+			keyword: props.keyword,
+		};
+	}
+
+	/**
+	 * Checks the keyword input for comma-separated words
+	 *
+	 * @param {String} keywordText The text of the input
+	 *
+	 * @returns {void}
+	 */
+	checkKeywordInput( keywordText ) {
+		let separatedWords = keywordText.split( "," );
+		if ( separatedWords.length > 1 ) {
+			this.setState( { showErrorMessage: true } );
+		} else {
+			this.setState( { showErrorMessage: false } );
+		}
+	}
+
+	/**
+	 * Displays the error message
+	 *
+	 * @param {String} input 		 The text of the input
+	 *
+	 * @returns {Element} ErrorText  The error message element
+	 */
+	displayErrorMessage( input = "" ) {
+		if ( this.state.showErrorMessage && input !== "" ) {
+			return (
+				<ErrorText>
+					<FormattedMessage
```

Comment:
After confirming with @IreneStr, we will no longer be using `react-intl` in the future. Since you are explicitly adding the `enzyme-react-intl` package to test this and we might forget to remove this later. Please remove this component and test package (but still use `role="alert"` of course!).